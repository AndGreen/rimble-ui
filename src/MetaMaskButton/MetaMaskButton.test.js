import MetaMaskButton from './'

describe('MetaMaskButton component sanity', () => {
  it('has name', () => {
    expect(MetaMaskButton.displayName).toBe('MetaMaskButton')
  })

  it('matches default snapshot', () => {
    const component = shallow(<MetaMaskButton />)
    expect(component).toMatchSnapshot()
  })
})
