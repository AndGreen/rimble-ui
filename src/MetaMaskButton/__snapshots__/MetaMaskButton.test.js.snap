// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetaMaskButton component sanity matches default snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-gQORMK",
        "isStatic": false,
        "rules": Array [
          "
      > span {
        display: inherit;
        align-items: center;
      }
    
      > span > svg:first-child {
        display: block;
        height: ",
          [Function],
          ";
        width: ",
          [Function],
          ";
        margin-right: .5rem;
      }
    ",
          "
      & {
        color: white;
        background: black;
        box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
      }
      &:hover {
        background-color: rgba(0, 0, 0, 0.7);
      }
      &:active {
        background-color: rgba(0, 0, 0, 0.4);
      }
    ",
        ],
      },
      "displayName": "MetaMaskButton",
      "outline": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": undefined,
        "componentStyle": ComponentStyle {
          "componentId": "sc-eiqfgm",
          "isStatic": false,
          "rules": Array [
            "
      > span {
        display: inherit;
        align-items: center;
      }
    
      > span > svg:first-child {
        display: block;
        height: ",
            [Function],
            ";
        width: ",
            [Function],
            ";
        margin-right: .5rem;
      }
    ",
            "
      & {
        color: #333333;
        background: #FFFFFF;
        border: 1px solid #CCCCCC;
        box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
      }
      &:hover {
        background-color: #F3F2F2;
      }
      &:active {
        background-color: #E4E4E4;
      }
    ",
          ],
        },
        "displayName": "Styled(Styled(ProtoButton))",
        "render": [Function],
        "styledComponentId": "sc-eiqfgm",
        "target": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
        Symbol(Symbol.hasInstance): [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-gQORMK",
      "target": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
      Symbol(Symbol.hasInstance): [Function],
    }
  }
  forwardedRef={null}
/>
`;
